### DISCUSSION TITLE
AI deck & Light house: Wifi camera stream stops on connect

### DISCUSSION THREAD

**Discussion by TommiAux** [bitcraze/discussions#752] [Q&A]
Hello,

currently I am trying to run the [wifi-img-streamer](https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/other/wifi-img-streamer)  example, while flying with an attached light house deck. Unfortunately I am experiencing strange behavior and maybe someone else also made this observations.

Connecting to the wifi AP of the AI deck works fine and starting the example is also no problem. The video streams smoothly at this point. However, when connecting to the Crazyflie from the client, the stream freezes. Interestingly this only happens with an attached light house deck. Flying with a Flow deck works like a charm while streaming the camera feed.

I found [this topic](https://github.com/orgs/bitcraze/discussions/519) about the compatibility of the AI deck and light house.  But as far as my understanding goes, the AI deck does not rely on UART1 anymore?

The software used is up to date. I flashed the AI deck's ESP32 with the newest firmware and the Crazyflie firmware is taken from the current master branch. I also removed the console prints from the wifi-img-streamer example to take some load off the CPX, however, with no improvement.

Any ideas or advice is greatly appreciated :)
Tommi

**Comment by tobbeanton** [comment]
I tried this with the 2023.02 release and it worked for me. Could it be that the WiFi and Crazyradio channels are colliding?

Other problems could be a voltage dip during take-off that potentially resets the ESP32 and the LH-deck.

**Comment by TommiAux** [comment]
Thanks for your reply @tobbeanton and for trying it out. So today I gave it another try and I manged to get the stream while being connected to the Crazyflie. However, it doesn't work every time. In such cases I have to keep power-cycling the Crazyflie and starting over.

If the stream does not completely freeze, the FPS of the video stream drop significantly after connecting to the Crazyflie. But I guess this is due to the Crazyradio and wifi. Changing the Crazyradio channels didn't show any improvement.

Maybe there is also a problem with my computer. I am running a native Ubuntu 20.04 LTS on my laptop and did not have any problems so far, but I will try it out on another computer.

**Reply by tobbeanton** [reply]
As a simple test can you connect to the Crazyflie using the USB cable + cfclient instead. This would indicate if there is a WiFi - Crazyradio interference.
